src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -752,13 +752,25 @@ private function registerValidationConfiguration(array $config, ContainerBuilder
 
         $container->setParameter('validator.translation_domain', $config['translation_domain']);
 
-        list($xmlMappings, $yamlMappings) = $this->getValidatorMappingFiles($container);
-        if (count($xmlMappings) > 0) {
-            $validatorBuilder->addMethodCall('addXmlMappings', array($xmlMappings));
+        $files = array('xml' => array(), 'yml' => array());
+        foreach ($this->getValidatorMappingFiles($config, $container) as $file) {
+            if (!is_file($file)) {
```

Comment:
this check should be moved instead ``getValidatorMappingFiles`` to be done only for user-configured files, as others are already checked in the function.